### PR TITLE
Slightly improved screen coordinates calculation

### PR DESCRIPTION
**Pull request by chevtche**

### CODE REVIEW

File: livre/core/Render/RenderBrick.cpp [hunk 1]
```diff
@@ -105,30 +105,23 @@ void RenderBrick::getScreenCoordinates( const Frustum& frustum,
                     continue;
                 in.normalize();
 
-                const double perspCorr = 1.0 / in.w();
+                Vector3d in3( in );
```
Comment (bilgili) [inline livre/core/Render/RenderBrick.cpp]: At least put a comment on top of it that homogenization is done on construction time. This is so implicit.

File: livre/core/Render/RenderBrick.cpp [hunk 2]
```diff
@@ -105,30 +105,23 @@ void RenderBrick::getScreenCoordinates( const Frustum& frustum,
...
 
-                in[0] *= perspCorr;
-                in[1] *= perspCorr;
-                in[2] *= perspCorr;
-
-                /* Map x, y and z to range 0-1 */
-                in[0] = in[0] * 0.5 + 0.5;
-                in[1] = in[1] * 0.5 + 0.5;
-                in[2] = in[2] * 0.5 + 0.5;
+                in3 = in3 * 0.5 + 0.5;
```
Comment (bilgili) [inline livre/core/Render/RenderBrick.cpp]: kepp the comment

**Comment by chevtche** [comment]
Retest this please

**Comment by chevtche** [comment]
Can someone merge this?

**Pull request merged**